eng/Version.Details.xml

```diff
@@ -1,62 +1,86 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.6.20278.9">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>567a7a2bb62e180ddb224ba820297e48b1879df1</Sha>
+    <Dependency Name="Microsoft.CodeAnalysis.Razor" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Razor.Internal.Transport" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="5.0.0-preview.6.20278.9">
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X" Version="5.0.0-preview.6.20278.12">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>a799bf68e45c63e62060a7916ebb6d4f87a9aa1b</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.6.20277.12" CoherentParentDependency="Microsoft.CodeAnalysis.Razor">
```

Comment:
This is very weird and is not coherent with the dotnet/aspnetcore 5.0.0-preview.6.20278.12 build. We also don't want to downgrade these dependencies. Suspect you need to run `darc update-dependencies --coherency-only` after we get dotnet/aspnetcore#1947 in.

@mmitche @wtgodbe the dotnet/aspnetcore is using dotnet/runtime dependencies at 5.0.0-preview.6.20271.10 at the a799bf68e45c63e62060a7916ebb6d4f87a9aa1b commit. Why didn't `CoherentParentDependency` do the "correct" thing and back up all that way? We don't really want to downgrade at all but that at least would have been coherent.

Reply:
It did the expected thing when I ran it.

Reply:
You mean it downgraded to 5.0.0-preview.6.20271.10?

Reply:
Oops, I pointed to the wrong dependency update PR.

@JunTaoLuo only just added an arc updating dotnet/aspnetcore directly from dotnet/runtime. I triggered it moments ago. This PR won't be ready to go (really, the dependencies won't be coherent without downgrading) until dotnet/aspnetcore#22370 and the subsequent official build and Maestro++ update are done.

Reply:
@JunTaoLuo you've got lots of approval here but please don't merge until you're able to update the dependencies in this branch correctly.

Reply:
Yep, it downgraded locally.  What's the darc version that got used to do that update? Seems unlikely that would affect things but could be.

Reply:
Locally for me, darc doesn't seem to do any coherency updates, but it could be because aspnetcore hasn't ingested newer runtime versions yet. I'm waiting to confirm that once the runtime update in aspnetcore is merged.

Reply:
Triggered a subscription update and looks like the latest commit shows that updates are working as expected.